Silicon/CommonSocPkg/Library/WatchDogTimerLib/WatchDogTimerLib.c

```diff
@@ -98,14 +98,14 @@ WdtDisable (
 
 
 /**
-  Clear WDT timer flags.
+  Clear WDT TCC flags in scratchpad
 
-  @param[in] Flags             The timer flags that would be cleared.
+  @param[in] Flags             The scratchpad flags that would be cleared.
 
 **/
 VOID
 EFIAPI
-WdtClearFlags (
+WdtClearTccScratchpad (
```

Comment:
WdtDogTimerLib is for common usage, and it should not be specific to TCC.
Please keep the API interface generic and use the flag to indicate TCC use cases.

Reply:
The original implementation (line 115) only takes care of bit fields within WDT_FLAG_MASK, which is to clear scratchpad and no other bit fields. Therefore, WdtClearScratchpad should be more precise. (I will rename it to WdtClearScratchpad () in the next revision)

Reply:
will be more generic, with a mask (so no need to be for scratchpad only)